theme.user.css

```diff
@@ -233,30 +242,51 @@
     color: var(--color-status-passed-text);
   }
 
+  .build-page-refresh-on .panel-warning,
   .build-page-refresh-off .panel-warning {
     border-color: var(--color-status-warn-border);
   }
 
+  .build-page-refresh-on .panel-warning .panel-heading,
+  .build-page-refresh-on .build-pipeline-job.build-pipeline-state-running,
   .build-page-refresh-off .panel-warning .panel-heading,
   .build-page-refresh-off .build-pipeline-job.build-pipeline-state-running {
     background-color: var(--color-status-warn-bg);
     border-color: var(--color-status-warn-border-light);
     color: var(--color-status-warn-text);
   }
 
+  .build-page-refresh-on .build-panel.panel-danger,
   .build-page-refresh-off .build-panel.panel-danger {
     border-color: var(--color-status-failed-border);
   }
 
+  .build-page-refresh-on .panel-danger .panel-heading,
+  .build-page-refresh-on .build-pipeline-job.build-pipeline-state-failed,
+  .build-page-refresh-on .build-pipeline-job.build-pipeline-state-failed-soft,
+  .build-page-refresh-on
```

Comment:
I wonder if it would be possible to use `:is()` for this like so:

```css
:is(.build-page-refresh-on, .build-page-refresh-off)
```

That could help reduce duplication.